### PR TITLE
Update rancher related docs and setting config

### PR DESCRIPTION
**Pull request by guangbochen**
- Update rancher related docs 
- set `auto-disk-provision-paths` setting as experimental

### CODE REVIEW

File: docs/networking/harvester-network.md [hunk 1]
```diff
@@ -19,15 +19,16 @@ Moreover, the Harvester UI integrates the [harvester-network-controller](https:/
...
-- [VLAN](#vlan-network)
+- [Management Network](#management-network)
+- [VLAN Network](#vlan-network)
 
 
 ## Management Network
 
-Harvester uses [flannel](https://github.com/flannel-io/flannel) CNI as its default management network. It is a built-in network that can be used directly from the cluster. However, the management network IP is not persisted and will be changed after a VM reboot.
+Harvester uses [canal](https://projectcalico.docs.tigera.io/getting-started/kubernetes/flannel/flannel) as its default management network. It is a built-in network that can be used directly from the cluster. 
+By default, its management network IP can only be accessed within the cluster nodes, and the management network IP will not remain unchanged after the VM reboot.
```
Comment (gitlawr) [inline docs/networking/harvester-network.md:29]: `its management network IP` => `management network IP of a VM`

File: docs/rancher/rke1-cluster.md [hunk 2]
```diff
@@ -48,6 +59,8 @@ Users can create an RKE1 Kubernetes cluster from the **Cluster Management** page
 1. Enter **Name Prefix** (required).
 1. Enter **Template** (required).
 1. Select **etcd** and **Control Plane** (required).
+1. On the **Cluster Options** config `Cloud Provider` to `Harvester` if you want to use the Harvester [Cloud Provider](/rancher/cloud-provider) and [CSI Diver](/rancher/csi-driver).
```
Comment (gitlawr) [inline docs/rancher/rke1-cluster.md:62]: s/config/configure/

**Comment by guangbochen** [comment]
updated, thanks.

File: docs/rancher/rke1-cluster.md [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
-      - Harvester RKE1 node driver is in tech preview.
       - VLAN network is required for Harvester node driver.
 
+When you create a Kubernetes cluster hosted by the Harvester infrastructure, [node templates](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/#node-templates) are used to provision the cluster nodes. These templates use Docker Machine configuration options to define an operating system image and settings/parameters for the node.
+
+Node templates can use cloud credentials to access the credential information required to provision nodes in the infrastructure providers. The same cloud credential can be used by multiple node templates. By using a cloud credential, you do not have to re-enter access keys for the same cloud provider. Cloud credentials are stored as Kubernetes secrets.
+
+You can create cloud credentials in two contexts:
+
+- [During creation of a node template](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/#node-templates) for a cluster.
```
Comment (nunix) [inline docs/rancher/rke1-cluster.md:16]:
```suggestion
- [During the creation of a node template](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/rke-clusters/node-pools/#node-templates) for a cluster.
```

File: docs/rancher/rke1-cluster.md [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
+- In the User Settings
```
Comment (nunix) [inline docs/rancher/rke1-cluster.md:17]:
```suggestion
- In the User Settings page
```

**Review by gitlawr — APPROVED** [review]
LGTM

File: docs/rancher/rke1-cluster.md [hunk 5]
```diff
@@ -1,13 +1,23 @@
...
+
+All cloud credentials are bound to the user profile of who created it. They cannot be shared across users.
```
Comment (nunix) [inline docs/rancher/rke1-cluster.md:19]:
```suggestion
All `cloud credentials` are bound to your user profile and cannot be shared with other users.
```

**Review by nunix — COMMENTED** [review]
Standardized the versions and rephrased to active voice

**Pull request merged**